src/index.ts

```diff
@@ -719,6 +751,53 @@ export class Optional<S, A> {
   readonly _tag: 'Optional' = 'Optional'
   constructor(readonly getOption: (s: S) => Option<A>, readonly set: (a: A) => (s: S) => S) {}
 
+  /**
+   * @example
+   * import { Optional } from 'monocle-ts'
+   *
+   * interface Phone {
+   *   number: string
+   * }
+   * interface Employment {
+   *   phone?: Phone
+   * }
+   * interface Info {
+   *   employment?: Employment
+   * }
+   * interface Response {
+   *   info?: Info
+   * }
+   *
+   * const numberFromResponse = Optional.fromNullablePath<Response>()(['info', 'employment', 'phone', 'number'])
+   *
+   * const response1: Response = {
+   *   info: {
+   *     employment: {
+   *       phone: {
+   *         number: '555-1234'
+   *       }
+   *     }
+   *   }
+   * }
+   * const response2: Response = {
+   *   info: {
+   *     employment: {}
+   *   }
+   * }
+   *
+   * numberFromResponse.getOption(response1) // some('555-1234')
+   * numberFromResponse.getOption(response2) // none
+   *
+   * @since 2.0.1
+   */
+  static fromNullablePath<S>(): OptionalFromPath<S> {
```

Comment:
I'd rename this function to `fromPath` since it works even if all props are required

Reply:
Make sense. 
Renamed to `fromPath` and updated the PR title accordingly